src/App.jsx

```diff
@@ -1,47 +1,43 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
 
-import { useState } from 'react';
-import { Courses, CreateCourse, Header } from './components';
+import {
+	CourseInfo,
+	Courses,
+	CreateCourse,
+	Login,
+	Registration,
+} from './components';
 
 import { mockedAuthorsList, mockedCoursesList } from './helpers';
-import { Container } from 'react-bootstrap';
-
-const RENDERING_OPS = {
-	CREATE_COURSE: true,
-	COURSE_CARDS: false,
-};
+import { Route, Routes } from 'react-router-dom';
+import axios from 'axios';
+import './i18n';
 
 function App() {
-	const [createCourseClicked, setCreateCourseClicked] = useState(false);
+	axios.defaults.baseURL = 'http://localhost:4000';
 
-	const renderConditionally = () => {
-		switch (createCourseClicked) {
-			case RENDERING_OPS.CREATE_COURSE:
-				return <CreateCourse authorsList={mockedAuthorsList} />;
-			case RENDERING_OPS.COURSE_CARDS:
-				return (
+	return (
+		<Routes>
+			<Route path='/registration' element={<Registration />} />
+			<Route path='/login' element={<Login />} />
+			<Route
+				path='/courses/:courseId'
+				element={<CourseInfo courses={mockedCoursesList} />}
+			/>
+			<Route
+				path='/courses'
+				element={
 					<Courses
-						onCreateCourseBtnClick={onCreateCourseBtnClicked}
 						coursesList={mockedCoursesList}
 						authorsList={mockedAuthorsList}
 					/>
-				);
-			default:
-				console.error('Error in conditional rendering');
-		}
-	};
-
-	const onCreateCourseBtnClicked = (isClicked) => {
-		setCreateCourseClicked(isClicked);
-	};
-
-	return (
-		<div>
-			<Header />
-			<Container className='border border-primary'>
-				{renderConditionally()}
-			</Container>
-		</div>
+				}
+			/>
+			<Route
+				path='/courses/add'
+				element={<CreateCourse authorsList={mockedAuthorsList} />}
+			/>
+		</Routes>
```

Comment:
It would be nice to add automatic redirect from non-existing page to home page (courses, for example). `Navigate` from 'react-router-dom' can help with that.

```suggestion
			<Route path='*' element={<Navigate to={Routes.Courses} replace />} />
		</Routes>
```